repository/src/test/java/org/alfresco/repo/exporter/ExporterComponentTest.java

```diff
@@ -153,15 +156,62 @@ public void testExport()
         parameters.setExportFrom(location);
 //        parameters.setExcludeAspects(new QName[] { ContentModel.ASPECT_AUDITABLE });
 //        parameters.setExcludeChildAssocs(new QName[] { ContentModel.ASSOC_CONTAINS });
-        
+

```

Comment:
Do we have any checks to verify that it is working as expected now that you are using a new approach?